app/views/questionnaire.py

```diff
@@ -361,6 +348,21 @@ def update_questionnaire_store_with_form_data(questionnaire_store, location, ans
         questionnaire_store.completed_blocks.append(location)
 
 
+def _format_answer_values(answer_value):
+
+    is_day_month_year = 'day' in answer_value and 'month' in answer_value and 'year' in answer_value
+    is_month_year = 'day' not in answer_value and 'year' in answer_value and 'month' in answer_value
+    is_time_input = 'hours' in answer_value or 'mins' in answer_value
+
+    if is_day_month_year and answer_value['day'] and answer_value['month']:
+        return "{:02d}/{:02d}/{}".format(int(answer_value['day']), int(answer_value['month']), answer_value['year'])
```

Comment:
Bit late in the day but why aren't we using ISO 8601 dates? I wonder if we should use ISO 8601 duration?